tests/testsuite/cargo_command.rs

```diff
@@ -262,9 +262,8 @@ fn find_closest_dont_correct_nonsense() {
         .cwd(&paths::root())
         .with_status(101)
         .with_stderr(
-            "[ERROR] no such subcommand: \
-                        `there-is-no-way-that-there-is-a-command-close-to-this`
-",
+            "[ERROR] no such subcommand: `there-is-no-way-that-there-is-a-command-close-to-this`\n
+<tab>View all installed commands with `cargo --list`",
```

Comment:
Both errors are mixing newlines with `\n`, making it confusing to spot check how this looks to users.  Could we consistently use newlines for it?